README.md

```
@@ -1,253 +1,307 @@
+
+## Quick Tenant Configuration
+
+Configuration database scripts are provided to generate a tenant named `Test-Hospital`. These scripts will load most of the service configurations needed to run Link Cloud. To be capable of performing resource acquisition, a FHIR endpoint will need to be manually added (detailed below). Here are the steps to run the scripts:
+
+1. Open SQL Server Management Studio and connect to your SQL Server Docker container (Defaulted to port 1433).
+
+2. Open and run `\link-cloud\Scripts\load-sql-data.sql`.
+
+3. To properly query for FHIR resources, update the FHIR endpoints in the SQL `link-dataacquisition` database:
+
+```   
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)